richie/js/utils/filters/getFilterFromState.ts

```diff
@@ -48,12 +48,12 @@ export function getFilterFromState(
     // We don't have the facets yet or something broke upstream: provide some filtering
     // capabilities anyway (without counts, as we can't generate those)
     if (!facets_[resourceName] || !Object.keys(facets_[resourceName]).length) {
-      return values(state.resources[resourceName]!.byId || {}).map(
-        organization => ({
-          humanName: organization.name,
-          primaryKey: String(organization.id),
-        }),
-      );
+      return values(state.resources[resourceName]!.byId)
+        .filter(organization => !!organization)
+        .map(organization => ({
+          humanName: organization!.name,
```

Comment:
Type assertion operator (https://www.typescriptlang.org/docs/handbook/advanced-types.html)

It basically tells the compiler that `organization` is not `null` or `undefined`. We know it because we ran a check just above, but the compiler was not smart enough to understand it